spyder/plugins/editor/panels/classfunctiondropdown.py

```diff
@@ -29,8 +29,8 @@ class ClassFunctionDropdown(Panel):
         The editor to act on.
     """
 
-    def __init__(self, editor):
-        super(ClassFunctionDropdown, self).__init__(editor)
+    def __init__(self):
+        super(ClassFunctionDropdown, self).__init__()
```

Comment:
```suggestion
        super().__init__()
```

We can simplify this call because now we only support Python 3.